### PR TITLE
Creation of Admin Choices

### PR DESCRIPTION
**Pull request by mapajohnedel**
PR description: Creation of Admin Choices
Content: Admin choices and answer
[Commands]
rails db:migrate:reset db:seed
rails server

[Pre-conditions]
Login as admin 
email: example@railstutorial.org
password: password

Go to manage categories choose a category and click add word
it should redirect to the index page of word with title and content

Now click add word
It should allow users to add words, choices, and the correct answer.  user could also update and delete words

[Expected Output]
Admin user can CRUD words and assign choices and correct answer
			
[Notes]
Validation is still not working for the choices, which user can choose more than 1 correct answer. I will work on that tomorrow for the polish of admin side.

### CODE REVIEW

**Review by inahsanjuan — CHANGES_REQUESTED** [review]
> **`Resolved`**

@johnedelmapa 
Could not proceed checking, this error occurs once I click `Add word` button
(I already tried to reset migrations and seeders)

image

File: app/views/admin/words/index.html.erb [hunk 1]
```diff
@@ -18,7 +18,7 @@
                 <% @words.each do |word| %>
                 <tr>
                     <td><%= word.content %></td>
-                    <td><%= word.correct_ans%></td>
+                    <td><%= word.correct_ans.content%></td>
```
Comment (inahsanjuan) [inline app/views/admin/words/index.html.erb]: @johnedelmapa This change is actually same with the implementation before.
The error is occurring only on first category regarding on the data that is related with the `word` object. Please investigate more on that.

**Review by inahsanjuan — CHANGES_REQUESTED** [review]

File: app/views/admin/words/index.html.erb (hunk 1 above)
Comment (mapajohnedel) [reply app/views/admin/words/index.html.erb]: Is there still a error ?

Comment (inahsanjuan) [reply app/views/admin/words/index.html.erb]: @johnedelmapa Yes, as you can see my comment before is not resolved yet

Comment (mapajohnedel) [reply app/views/admin/words/index.html.erb]: everything works perfectly on my side, no errors. I have no idea whats the error, but I added some line of codes please check. please run rails db:migrate:reset first

Comment (mapajohnedel) [reply app/views/admin/words/index.html.erb]: and db:seed

File: db/seeds.rb [hunk 2]
```diff
@@ -34,18 +34,3 @@
 following.each { |followed| user1.follow(followed) }
 followers.each { |follower| follower.follow(user1) }
 
-49.times do |n|
-  title  = Faker::Book.title
-  des = "Lorem Ipsums"
-
-  Category.create!(title:  title,
-                  description: des)
-end
```
Comment (inahsanjuan) [inline db/seeds.rb]: @johnedelmapa Don't remove category seeders. This is very helpful for testing

**Review by inahsanjuan — CHANGES_REQUESTED** [review]
@johnedelmapa 
OK
- Choice CRUD functionality
- Choosing correct answer

Comment
- Minor seeder update

________

> Additional info regarding error a while ago:
The reason for that is you have added `word seeder` but you did not provide any data for `choices`, that's why whenever `.content` method is being called, it returns error because of the missing data

> Problem mentioned above was solved by removal of the `word seeder`, but you can actually provide seeder for choice so you can still have persistent data.

File: db/seeds.rb (hunk 2 above)
Comment (mapajohnedel) [reply db/seeds.rb]: Noted thank you for the info.

**Pull request merged**